app/graphql/query/tripsCollection.graphql

```diff
@@ -0,0 +1,31 @@
+query tripsCollection($user_id: BigInt!) {
+  tripsCollection(filter: { user_id: { eq: $user_id } }) {
+    edges {
+      node {
+        id
+        uuid
+        title
+        date_from
+        date_to
+        invitationsCollection {
+          edges {
+            node {
+              users {
+                id
+                name
+              }
```

Comment:
relay style graphql schema, nice !
And I'm just wondering how to restrict query depth, since we do not have to execute deeply nested query for performance and security. But as long as I see supabase docs, I didn't find how supabase handle with it...

Reply:
Not this PR for sure, but we're going to handle with this sooner or later...

Reply:
>  I didn't find how supabase handle with it...

It can handle with `pg_graphql` 👍 
https://supabase.github.io/pg_graphql/

> Not this PR for sure, but we're going to handle with this sooner or later...

You're right, it's not for this PR, but it's definitely something we'll need to address soon. Let's add it on our task board for upcoming task. 👍🏻

Reply:
Ohh cool !